.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Go
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+  release:
+
+jobs:
+  build:
+    name: CI
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Copyright notice
+      run: |
+        .scripts/check-copyright-notice.sh
+        .scripts/check-notice-authors.sh origin/master
+
+    - name: Dependencies
+      run: |
+        pip3 install slither-analyzer
```

Comment:
Please try to use the slither docker image: https://github.com/crytic/slither#using-docker

There is also a problem with this line, which you can see in the perun-network ci: https://github.com/perun-network/perun-eth-contracts/runs/2004951308